www/_data/redirects.yml

```diff
@@ -166,3 +166,5 @@ docs-global:
     - {old: "guide_plugin-development_blackberry10_index.md.html", new: "guide/plugin-development/blackberry10/index.html"}
     - {old: "guide_privacy_index.md.html", new: "guide/privacy/index.html"}
     - {old: "guide_upgrading_windows-8_index.md.html", new: "guide/upgrading/windows-8/index.html"}
+    - {old: "dev/guide/platforms/ios/tools.html", new: "dev/guide/platforms/ios/index.html"}
+    - {old: "dev/guide/platforms/ios/config.html", new: "dev/config_ref/index.html"}
```

Comment:
Two things:
1. These are version-specific, not global: please place them in the `docs` section in this file; the difference is described [here][redirects]
2. Are these redirects necessary, or could the files stay where they are?

[redirects]: https://github.com/apache/cordova-docs#redirects